### DISCUSSION TITLE
empty text and no tables from PDF which shows content in Adobe Acrobat Reader

### DISCUSSION THREAD

**Discussion by woklenk** [pymupdf/PyMuPDF#3787] [Looking for help]
Hello, I try to get the text and tables from PDF documents, not only this one here from JP authority
 https://www.mlit.go.jp/report/press/content/001750315.pdf
which are shown by Adobe Acrobat Reader.  But PyMuPDF shows me a empty text and no tables.

What is going wrong? Can somebody help?

Regards
Wolfgang
[001750315[1].pdf](https://github.com/user-attachments/files/16636678/001750315.1.pdf)

**Comment by JorjMcKie** [comment]
There exists no text on the page.
What you se are drawings / vector graphics, constructed in a way that only **_looks_** like text.
Your only option is OCR.

**Comment by woklenk** [comment]
ok and thank you. I had already suspected something like that, but I couldn't extract an image / vector graphics either.....

**Reply by JorjMcKie** [reply]
Oh, you can extract vector graphics of course: every character is written as one.

**Comment by woklenk** [comment]
ok, thanks, I have to try this again
